src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/SelectInstanceTypeSection.tsx

```diff
@@ -20,7 +20,11 @@ const SelectInstanceTypeSection: React.FC<SelectInstanceTypeSectionProps> = ({
   setSelectedInstanceType,
 }) => {
   const handleSelect = (category: InstanceTypeCategory, size: InstanceTypeSize) => {
-    setSelectedInstanceType({ category, size, name: `${seriesDetails[category].prefix}.${size}` });
+    setSelectedInstanceType({
+      category,
+      size,
+      name: `${categoryDetailsMap[category].prefix}.${size}`,
```

Comment:
Maybe I'd better use `${categoryDetailsMap[category]?.prefix}.${size}`, as it's already used like this in `getInstancetypeString`, or are we sure there's gonna always be some `prefix`? I am not sure :)

Reply:
Done!